src/Converter.re

```diff
@@ -222,6 +226,8 @@ let rec converterIsIdentity = (~toJS, converter) =>
     }
 
   | RecordC(_) => false
+  | TupleC(innerTypesC) =>
+    innerTypesC |> List.for_all(converterIsIdentity(~toJS))
```

Comment:
Mostly the same as the [`ObjectC` case above](https://github.com/cristianoc/genType/pull/71/files#diff-b8ec785735c28040e738985bd4cce5e6L212), but didn't use the `OptionC` switch because I didn't think it was necessary (doesn't that case get process naturally when the recursion continues?)

Reply:
Option has no special role here, while it does for objects. So this is perfect.